sdk/src/openssl/ed_signer.rs

```diff
@@ -35,14 +36,15 @@ pub struct EdSigner {
 }
 
 impl ConfigurableSigner for EdSigner {
+    #[cfg(feature = "file_io")]
```

Comment:
Not crucial for this PR, but I wonder if we could move the `from_files` into a default implementation in `ConfigurableSigner`? Seems like a lot of boilerplate in these files.

Reply:
good idea

Reply:
I made this change